### ISSUE TITLE
/usr/bin/qucsator: symbol lookup error

### ISSUE THREAD

**Issue by HE1010** [Qucs/qucs#804]
QUCS is not working in my installation. It always aborts with the following message:

Output:
-------
Starting new simulation on Fr. 15. Jun 2018 at 09:09:54:264
creating netlist... done.
Errors occurred during simulation on Fr. 15. Jun 2018 at 09:09:54:474
Aborted.

Errors:
-------
/usr/bin/qucsator: symbol lookup error: /usr/bin/qucsator: undefined symbol: _ZN4qucs6module22registerDynamicModulesEPcSt4listISsSaISsEE

No hint how to solve, very disappointing. I probably will uninstall and choose  another program.

**Comment by in3otd** [comment]
> how did you install /usr/bin/qucsator? what system do you use?

yep, did you install a package intended for one distribution on another distribution?
It may be that the simulator library ended up in an unexpected place (`/usr/local/lib` instead of `/user/lib`?) . Do you have the variable `LD_LIBRARY_PATH` set?

>  I hope that gnucap will finally supersede qucsator.

but this may not solve issues related to misplaced libraries :grin:
